napalm_ansible/modules/napalm_install_config.py

```diff
@@ -289,6 +289,8 @@ def main():
     try:
         if get_diffs:
             diff = device.compare_config().encode('utf-8')
+            if type(diff) is bytes:
```

Comment:
I think `if isinstance(diff, bytes)` would be better than comparing types.

Reply:
Ok, I can fix that and re-commit.